staticfiles/admin/css/autocomplete.4a81fc4242d0.css

```diff
@@ -0,0 +1,275 @@
+select.admin-autocomplete {
```

Comment:
https://stackoverflow.com/questions/41558258/should-i-add-django-admin-static-files-to-my-git-repo/41558803

Please take a look if it possible to avoid committing this staff